python/GafferUI/EditScopeUI.py

```diff
@@ -409,6 +430,17 @@ def __menuDefinition( self ) :
 			},
 		)
 
+		if self.__globalEditTargetPlug() is not None :
+			result.append( "/__FollowDivider__", { "divider" : True, "label" : "Options" } )
+			result.append(
+				"/Follow Global Edit Target",
+				{
+					"command" : functools.partial( self.__connectPlug, self.__globalEditTargetPlug() ),
```

Comment:
Another spot where we need a WeakMethod, or could just use `self.getPlug.setInput` as the target for the `partial`.

I don't have steps to reproduce it, but I got this error on shutdown during testing, and I would hope that tidying these up would fix it :

```
ERROR : Emitting signal : Traceback (most recent call last):
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/Gaffer/WeakMethod.py", line 65, in __call__
ERROR :     return self.__method( s, *args, **kwArgs )
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/GafferUI/EditScopeUI.py", line 201, in __plugInputChanged
ERROR :     self.__acquireContextTracker()
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/GafferUI/EditScopeUI.py", line 219, in __acquireContextTracker
ERROR :     self.__updateMenuButton()
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/GafferUI/EditScopeUI.py", line 262, in __updateMenuButton
ERROR :     self.__menuButton.setText( editScope.getName() if editScope is not None else "Source" )
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/GafferUI/MenuButton.py", line 85, in setText
ERROR :     GafferUI.Button.setText( self, text )
ERROR :   File "/home/john/dev/build/gaffer-1.5/python/GafferUI/Button.py", line 95, in setText
ERROR :     self._qtWidget().setText( text )
ERROR : RuntimeError: Internal C++ object (PySide2.QtWidgets.QPushButton) already deleted.
```

Reply:
WeakMethod added in 5ac52ccb73c2f90383c6be75ae8cfb3c3bb3ddb7. I haven't spotted any `Internal C++ object (PySide2.QtWidgets.QPushButton) already deleted` while testing, so it may have done the trick.